stacks/root.yml

```diff
@@ -925,6 +929,44 @@ Resources:
           Value: !Ref AWS::StackId
       TemplateURL: !Join ["", ["http://s3", !If [IsUsEast1, "", "-"], !If [IsUsEast1, "", !Ref "AWS::Region"], ".amazonaws.com/", "Fn::ImportValue": !Sub "${InfrastructureStorageStackName}-InfrastructureSourceBucket", "/", !Ref InfrastructureGitCommit, "/stacks/proxy.prx.org.yml"]]
       TimeoutInMinutes: "5"
+  FeederStack:
```

Comment:
Feeder stack?

Reply:
Also we shouldn't be adding any new application stacks directly to the root stack. They should all be getting added to nested stacks (like the serverless or static-sites one that already exist).

Reply:
Hm. Not really. `legacy-support`?

Reply:
I guess ideally it would be something that we expect to add other things to in the future, so we don't end up with lots of nested stacks that only have one thing in them. So maybe something more generic than that?